pkg/kubelet/kubelet.go

```diff
@@ -2202,19 +2208,23 @@ func (kl *Kubelet) updateRuntimeUp() {
 
 	s, err := kl.containerRuntime.Status()
 	if err != nil {
-		klog.Errorf("Container runtime sanity check failed: %v", err)
+		klog.ErrorS(err, "Container runtime sanity check failed")
 		return
 	}
 	if s == nil {
-		klog.Errorf("Container runtime status is nil")
+		klog.ErrorS(nil, "Container runtime status is nil")
 		return
 	}
 	// Periodically log the whole runtime status for debugging.
-	klog.V(4).Infof("Container runtime status: %v", s)
+	klog.V(4).InfoS("Container runtime status", "status", s)
 	networkReady := s.GetRuntimeCondition(kubecontainer.NetworkReady)
 	if networkReady == nil || !networkReady.Status {
-		klog.Errorf("Container runtime network not ready: %v", networkReady)
-		kl.runtimeState.setNetworkState(fmt.Errorf("runtime network not ready: %v", networkReady))
+		if networkReady != nil {
+			klog.ErrorS(nil, "Container runtime network not ready", "networkReady", networkReady)
```

Comment:
@ehashman Was there going to be some support for nil handling in the library?

Reply:
I don't know why this wouldn't be safe, it's no different than the old log entry handling. `nil` just gets sprintf'd: https://github.com/kubernetes/klog/blob/407242c0258f5d907891f8797887b71a1c87e7b4/klog.go#L801-L836

It's not having `.String()` called on it directly anywhere.

Reply:
@ehashman done